integrated_channels/integrated_channel/migrations/0028_alter_contentmetadataitemtransmission_unique_together.py

```diff
@@ -17,24 +17,44 @@ class Migration(migrations.Migration):
             ),
         ]
     else:
-        operations = [
-            migrations.SeparateDatabaseAndState(
-                state_operations=[
-                    migrations.AlterUniqueTogether(
-                        name='contentmetadataitemtransmission',
-                        unique_together={('integrated_channel_code', 'plugin_configuration_id', 'content_id')},
-                    ),
-                ],
-                database_operations=[
-                    migrations.RunSQL(sql="""
-                        CREATE UNIQUE INDEX content_transmission_channel_code_plugin_id_content_id_unique
-                        ON integrated_channel_contentmetadataitemtransmission (integrated_channel_code, plugin_configuration_id, content_id)
-                        ALGORITHM=INPLACE LOCK=NONE
+        if 'postgresql' in db_engine:
```

Comment:
Same nit about using elif to reduce indentation here.

Reply:
agree.